### PR TITLE
I added retry logic to the New-AppPool function.

### PR DESCRIPTION
**Pull request by bskeen**

### CODE REVIEW

File: common/Fabric-Install-Utilities.psm1 [hunk 1]
```diff
@@ -84,11 +84,27 @@ function New-AppRoot($appDirectory, $iisUser){
...
+                $appPool = New-WebAppPool $appName -ErrorAction Stop
+                $successful = $TRUE
+            }
+            catch {
+                $failCount = $failCount + 1
+                if ($failCount -gt 3) {
+                    throw
+                }
+
+                Write-Host "Retry #$failCount"
```
Comment (michaelvidal24) [inline common/Fabric-Install-Utilities.psm1]: Can we make this message more specific, like "failed to create appPool $appName $failCount times , retrying..."

File: common/Fabric-Install-Utilities.psm1 [hunk 2]
```diff
@@ -84,11 +84,27 @@ function New-AppRoot($appDirectory, $iisUser){
 }
 
 function New-AppPool($appName, $userName, $credential){
-    Set-Location IIS:\AppPools
+    Set-Location "IIS:\AppPools"
     if(!(Test-Path $appName -PathType Container))
     {
         Write-Console "AppPool $appName does not exist...creating."
-        $appPool = New-WebAppPool $appName
+        $successful = $FALSE
```
Comment (michaelvidal24) [inline common/Fabric-Install-Utilities.psm1]: should we use $false here? I believe PowerShell is case sensitive...

File: common/Fabric-Install-Utilities.psm1 [hunk 3]
```diff
@@ -84,11 +84,27 @@ function New-AppRoot($appDirectory, $iisUser){
...
+        $failCount = 0
+        while (!$successful) {
+            try {
+                $appPool = New-WebAppPool $appName -ErrorAction Stop
+                $successful = $TRUE
```
Comment (michaelvidal24) [inline common/Fabric-Install-Utilities.psm1]: same comment as above.

File: common/Fabric-Install-Utilities.psm1 (hunk 1 above)
Comment (bskeen) [reply common/Fabric-Install-Utilities.psm1]: Thanks for catching that, I meant to fix that and forgot.

File: common/Fabric-Install-Utilities.psm1 (hunk 2 above)
Comment (bskeen) [reply common/Fabric-Install-Utilities.psm1]: I saw this syntax in a web page I was looking at while working on this. It seemed to work, but I can change it so it looks better.

**Review by michaelvidal24 — APPROVED** [review]

**Pull request merged**